core/src/include/Xios.hpp

```diff
@@ -146,7 +147,9 @@ class Xios : public Configured<Xios> {
     bool isEnabled;
 
     std::string clientId;
+    std::string _calendarType;
     std::string _contextId;
+    Duration _timestep;
```

Comment:
Unless this is an XIOS convention, I'd discourage leading underscores in nextSIM code, since we generally try to align with the WebKit coding standard (albeit imperfectly).

Reply:
Sure, addressed in 197d7473.